lib/ansible/module_utils/fortios.py

```diff
@@ -80,22 +90,27 @@ def backup(module, running_config):
         try:
             os.mkdir(backup_path)
         except:
-            module.fail_json(msg="Can't create directory {0} Permission denied ?".format(backup_path))
```

Comment:
Ansible coding guidelines allow up to 160 chars (which is enforced by CI)
Please revert these line-length changes it as
1) It's not needed
2) It makes the diff "noisy"